self-contained/bootstrap_helpers.sh

```diff
@@ -98,6 +98,7 @@ OS=$(sort -r /etc/*-release | gawk 'match($0, /^(ID_LIKE=(coreos)|ID=(.*))$/, a)
 OS_VERSION=$(sort -r /etc/*-release | gawk 'match($0, /^(VERSION_ID=(.*))$/, a) { print toupper(a[2] a[3]); exit }' | tr -d '"')
 UBUNTU_OS_NAME="UBUNTU"
 MARINER_OS_NAME="MARINER"
+AZURELINUX_OS_NAME="AZURELINUX"
```

Comment:
Does it make sense to make a list here instead?

Reply:
I am inclined on leaving the AZURELINUX_OS_NAME in the different places but using the `isMarinerOrAzureLinux` function when we need to reference both the OS's